[
    {
        "url": "https://api.github.com/repos/microsoft/onnxruntime/issues/comments/769633034",
        "html_url": "https://github.com/microsoft/onnxruntime/issues/6497#issuecomment-769633034",
        "issue_url": "https://api.github.com/repos/microsoft/onnxruntime/issues/6497",
        "id": 769633034,
        "node_id": "MDEyOklzc3VlQ29tbWVudDc2OTYzMzAzNA==",
        "user": {
            "login": "Pushpalathahareesh",
            "id": 24935208,
            "node_id": "MDQ6VXNlcjI0OTM1MjA4",
            "avatar_url": "https://avatars.githubusercontent.com/u/24935208?v=4",
            "gravatar_id": "",
            "url": "https://api.github.com/users/Pushpalathahareesh",
            "html_url": "https://github.com/Pushpalathahareesh",
            "followers_url": "https://api.github.com/users/Pushpalathahareesh/followers",
            "following_url": "https://api.github.com/users/Pushpalathahareesh/following{/other_user}",
            "gists_url": "https://api.github.com/users/Pushpalathahareesh/gists{/gist_id}",
            "starred_url": "https://api.github.com/users/Pushpalathahareesh/starred{/owner}{/repo}",
            "subscriptions_url": "https://api.github.com/users/Pushpalathahareesh/subscriptions",
            "organizations_url": "https://api.github.com/users/Pushpalathahareesh/orgs",
            "repos_url": "https://api.github.com/users/Pushpalathahareesh/repos",
            "events_url": "https://api.github.com/users/Pushpalathahareesh/events{/privacy}",
            "received_events_url": "https://api.github.com/users/Pushpalathahareesh/received_events",
            "type": "User",
            "site_admin": false
        },
        "created_at": "2021-01-29T07:31:03Z",
        "updated_at": "2021-01-29T07:31:03Z",
        "author_association": "NONE",
        "body": "Hi,\r\nPls check below link to install and set proper files path for cudaNN,related to your OS\r\nhttps://developer.nvidia.com/rdp/cudnn-download\r\n\r\nIam using onnxruntime for inferencing transformer model in windows 10 having GPU support.\r\nIn windows i have followed below link to set proper path for cudaNN files.Iam using CUDA 10.2\r\nhttps://towardsdatascience.com/installing-tensorflow-with-cuda-cudnn-and-gpu-support-on-windows-10-60693e46e781\r\n\r\nIam using below steps in windows 10:\r\nb)install cuDNN for CUDA 10.2\r\nc)unzip the cuDNN file and copy 3 files bin/cudnn64_7.dll,include/cudnn.h and lib/cudnn.lib to C:\\Program Files\\NVIDIA GPU Computing Toolkit\\CUDA\\v9.0\\bin,C:\\Program Files\\NVIDIA GPU Computing Toolkit\\CUDA\\v9.0\\include and C:\\Program Files\\NVIDIA GPU Computing Toolkit\\CUDA\\v9.0\\lib\\x64 directory\r\nd) check the path in environment variable if the path is set C:\\Program Files\\NVIDIA GPU Computing Toolkit\\CUDA\\v9.0\\bin and C:\\Program Files\\NVIDIA GPU Computing Toolkit\\CUDA\\v9.0\\libnvvp is set.\r\nd) run the following commands:\r\nimport onnxruntime\r\nprint(onnxruntime.get_device())#GPU\r\n\r\nregards,\r\npushpalatha M",
        "reactions": {
            "url": "https://api.github.com/repos/microsoft/onnxruntime/issues/comments/769633034/reactions",
            "total_count": 0,
            "+1": 0,
            "-1": 0,
            "laugh": 0,
            "hooray": 0,
            "confused": 0,
            "heart": 0,
            "rocket": 0,
            "eyes": 0
        },
        "performed_via_github_app": null
    },
    {
        "url": "https://api.github.com/repos/microsoft/onnxruntime/issues/comments/961535094",
        "html_url": "https://github.com/microsoft/onnxruntime/issues/6497#issuecomment-961535094",
        "issue_url": "https://api.github.com/repos/microsoft/onnxruntime/issues/6497",
        "id": 961535094,
        "node_id": "IC_kwDOCVq1mM45T9x2",
        "user": {
            "login": "dashesy",
            "id": 873905,
            "node_id": "MDQ6VXNlcjg3MzkwNQ==",
            "avatar_url": "https://avatars.githubusercontent.com/u/873905?v=4",
            "gravatar_id": "",
            "url": "https://api.github.com/users/dashesy",
            "html_url": "https://github.com/dashesy",
            "followers_url": "https://api.github.com/users/dashesy/followers",
            "following_url": "https://api.github.com/users/dashesy/following{/other_user}",
            "gists_url": "https://api.github.com/users/dashesy/gists{/gist_id}",
            "starred_url": "https://api.github.com/users/dashesy/starred{/owner}{/repo}",
            "subscriptions_url": "https://api.github.com/users/dashesy/subscriptions",
            "organizations_url": "https://api.github.com/users/dashesy/orgs",
            "repos_url": "https://api.github.com/users/dashesy/repos",
            "events_url": "https://api.github.com/users/dashesy/events{/privacy}",
            "received_events_url": "https://api.github.com/users/dashesy/received_events",
            "type": "User",
            "site_admin": false
        },
        "created_at": "2021-11-05T00:26:59Z",
        "updated_at": "2021-11-05T00:26:59Z",
        "author_association": "NONE",
        "body": "I only see this issue on onnxruntime. PyTorch is fine in finding the cuda libraries and cudnn so this is not related to that",
        "reactions": {
            "url": "https://api.github.com/repos/microsoft/onnxruntime/issues/comments/961535094/reactions",
            "total_count": 0,
            "+1": 0,
            "-1": 0,
            "laugh": 0,
            "hooray": 0,
            "confused": 0,
            "heart": 0,
            "rocket": 0,
            "eyes": 0
        },
        "performed_via_github_app": null
    },
    {
        "url": "https://api.github.com/repos/microsoft/onnxruntime/issues/comments/1101709552",
        "html_url": "https://github.com/microsoft/onnxruntime/issues/6497#issuecomment-1101709552",
        "issue_url": "https://api.github.com/repos/microsoft/onnxruntime/issues/6497",
        "id": 1101709552,
        "node_id": "IC_kwDOCVq1mM5BqsDw",
        "user": {
            "login": "stale[bot]",
            "id": 26384082,
            "node_id": "MDM6Qm90MjYzODQwODI=",
            "avatar_url": "https://avatars.githubusercontent.com/in/1724?v=4",
            "gravatar_id": "",
            "url": "https://api.github.com/users/stale%5Bbot%5D",
            "html_url": "https://github.com/apps/stale",
            "followers_url": "https://api.github.com/users/stale%5Bbot%5D/followers",
            "following_url": "https://api.github.com/users/stale%5Bbot%5D/following{/other_user}",
            "gists_url": "https://api.github.com/users/stale%5Bbot%5D/gists{/gist_id}",
            "starred_url": "https://api.github.com/users/stale%5Bbot%5D/starred{/owner}{/repo}",
            "subscriptions_url": "https://api.github.com/users/stale%5Bbot%5D/subscriptions",
            "organizations_url": "https://api.github.com/users/stale%5Bbot%5D/orgs",
            "repos_url": "https://api.github.com/users/stale%5Bbot%5D/repos",
            "events_url": "https://api.github.com/users/stale%5Bbot%5D/events{/privacy}",
            "received_events_url": "https://api.github.com/users/stale%5Bbot%5D/received_events",
            "type": "Bot",
            "site_admin": false
        },
        "created_at": "2022-04-18T19:53:53Z",
        "updated_at": "2022-04-18T19:53:53Z",
        "author_association": "NONE",
        "body": "This issue has been automatically marked as stale due to inactivity and will be closed in 7 days if no further activity occurs. If further support is needed, please provide an update and/or more details.\n",
        "reactions": {
            "url": "https://api.github.com/repos/microsoft/onnxruntime/issues/comments/1101709552/reactions",
            "total_count": 0,
            "+1": 0,
            "-1": 0,
            "laugh": 0,
            "hooray": 0,
            "confused": 0,
            "heart": 0,
            "rocket": 0,
            "eyes": 0
        },
        "performed_via_github_app": null
    }
]